src/main/java/racingcar/domain/Car.java

```diff
@@ -1,42 +1,55 @@
 package racingcar.domain;
 
+import racingcar.dto.CarRecord;
 import racingcar.strategy.MovingStrategy;
 
 import java.util.Objects;
 
 public class Car {
-    private static final int INIT_POSITION = 0;
+    public static final int INIT_POSITION = 0;
+    private final CarName name;
+    private final CarPosition position;
```

Comment:
```suggestion
    public static final int INIT_POSITION = 0;

    private final CarName name;
    private final CarPosition position;
```

상수와 인스턴스 변수 사이에는 보통 한 줄을 띄웁니다.
그리고, 상수의 접근 제어자가 `public`인 것은 실수일까요?